packages/neutrino/bin/start.js

```diff
@@ -2,16 +2,33 @@ const ora = require('ora');
 const debounce = require('lodash.debounce');
 const { start } = require('../src');
 const base = require('./base');
+const DashboardPlugin = require("webpack-dashboard/plugin");
+const { spawn } = require('child_process');
 
 module.exports = (middleware, args) => {
   const spinner = ora({ text: 'Building project' });
+  const argsClone = Object.assign({}, args);
+
+  if (argsClone.dashboard) {
+    argsClone.quiet = true;
+
+    middleware.push(neutrino => {
+      neutrino.config.plugin('dashboard').use(DashboardPlugin);
+    });
+
+    const child = spawn(`${__dirname  }/../../../node_modules/.bin/webpack-dashboard`, [], { stdio: 'inherit' });
+
+    process.on('exit', () => {
+      child.kill();
+    })
+  }
 
   return base({
     middleware,
-    args,
+    argsClone,
     NODE_ENV: 'development',
     commandHandler(config, neutrino) {
-      if (!args.start) {
+      if (!argsClone.start && !argsClone.dashboard) {
```

Comment:
Weird, I think there was a bug here, because that **should** say `args.quiet` instead of `args.start`! Can you change that to:

```js
if (!argsClone.quiet && !argsClone.dashboard) {
```